README.md

```diff
@@ -19,23 +20,22 @@ The individual components may also be installed from NuGet:
 - [Discord.Net.WebSocket](https://www.nuget.org/packages/Discord.Net.WebSocket/)
 - [Discord.Net.Webhook](https://www.nuget.org/packages/Discord.Net.Webhook/)
 
-### Unstable (MyGet)
-Nightly builds are available through our MyGet feed (`https://www.myget.org/F/discord-net/api/v3/index.json`).
+### Unstable / Nightly Build (via MyGet)
+Nightly builds are available through our [MyGet Feed](https://www.myget.org/F/discord-net/api/v3/index.json).
 
 ## Compiling
 In order to compile Discord.Net, you require the following:
-
-### Using Visual Studio
-- [Visual Studio 2017](https://www.microsoft.com/net/core#windowsvs2017)
+- [Visual Studio 2017+](https://www.microsoft.com/net/core#windowsvs2017) 
```

Comment:
Shouldn't this be updated to say VS2019? I thought we started using a few of C# 8.0's syntax sugars recently.